### ISSUE TITLE
portschecker container exited 

### ISSUE THREAD

**Issue by lazi444** [pi-node/instructions#81]
![1615025783474619101040493391614](https://user-images.githubusercontent.com/80153336/110203248-6c649c00-7e6d-11eb-84b4-0b59822f85c4.jpg)

**Comment by ihatejam** [comment]
It is normal for it to "exit" when the Consensus switch is turned on. Only one of the two containers can run at any given time - either the PortsChecker container (aka stellar-dummy) or the Consensus container (aka pi-consensus). 

When you switch on the Consensus container (with the red switch in the Node app), then the PortsChecker will be stopped. 
Conversely, when you go to check your ports, the Consensus container will be stopped and the PortsChecker will be started up.

**Comment by lazi444** [comment]
thank you very much

суб, 6. мар 2021. 22:34 ihatejam ***@***.***> је написао/ла:

**Comment by AbelKinkela** [comment]
@ihatejam @lazi444 does this mean mine is working fine and I dont have issues?
<img width="428" alt="image" src="https://user-images.githubusercontent.com/24853925/180593358-85bdc6ed-8e23-4d7d-b6c9-319b6cc73390.png">

**Comment by SAP2468** [comment]
yes your is fine working

**Comment by ravensheart6949** [comment]
can this be applied same with Ports Checker Container is absent?  
![image](https://user-images.githubusercontent.com/80130163/208230317-56f6df9b-fd55-4f4f-811c-67e32b627550.png)

**Comment by cheme75** [comment]
Yes, I’ve had to reinstall docker a few times and did not bother getting ports checker. The blockchain container loaded ran fine without it. As long as ports are open it will eventually get the blockchain container, first time it was exceptionally long time before it loaded.

**Comment by pjockey** [comment]
> Yes, I’ve had to reinstall docker a few times and did not bother getting ports checker. The blockchain container loaded ran fine without it. As long as ports are open it will eventually get the blockchain container, first time it was exceptionally long time before it loaded.

For me the "absent" has been showing for 3 weeks plus now.
